src/components/RepositoryEditor.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  ListItemSecondaryAction,
+  Avatar,
+  IconButton } from '@material-ui/core';
```

Comment:
`@material-ui/` import should each be their own import, directly from their file.

Reply:
Done.